src/main/java/org/myrobotlab/jme3/Jme3Servo.java

```diff
@@ -19,19 +18,32 @@
  */
 public class Jme3Servo extends Jme3Object implements VirtualServo {

```

Comment:
Why does this implement virtual servo? seems like it should implement ServoControl ...

Reply:
Yes, I think your right ... (head hurts sometimes)  - but Arduino's "HardwareServo" derives from VirtualServo - are you thinking this should be ServoControl too ?

The challenge with ServoControl is that it's really 2 interfaces - it represents what a user can do and it also represents how a ServoController can interface with it.

Reply:
The way I think of it, the only thing that needs to be virtual is the ServoController...  If you connect a real device to a virtual controller, the device becomes virtual by default... (at least for the servo case, and really anything that attaches to an arduino / i2c bus.)